modules/compatibility/src/test/java/org/apache/ignite/compatibility/persistence/SnapshotCompatibilityTest.java

```diff
@@ -142,19 +135,35 @@ public void setUp() {
     /** */
     @Test
     public void testSnapshotRestore() throws Exception {
+        assumeFalse("User must not restore both snapshots and cache dumps simultaneously", incSnp && cacheDump);
```

Comment:
"User must not restore both snapshots and cache dumps simultaneously" -> "Incremental snapshots for cache dump not supported"